cmd/goal/node.go

```diff
@@ -119,18 +120,44 @@ var nodeCmd = &cobra.Command{
 	},
 }
 
+func getMissingCatchPointLabel(genesis string) (label string, err error) {
+	URL := "https://algorand-catchpoints.s3.us-east-2.amazonaws.com/channel/" + genesis + "/latest.catchpoint"
```

Comment:
I think `algorand-catchpoints` also need to be extracted into a separate constant. fmt.Sprintf might look better